shared/src/commonTest/kotlin/co/touchlab/kampkit/TestAnnotations.kt

```diff
@@ -7,3 +7,22 @@ import kotlin.reflect.KClass
 expect annotation class RunWith(val value: KClass<out Runner>)
 expect abstract class Runner
 expect class AndroidJUnit4 : Runner
+
+@OptIn(ExperimentalMultiplatform::class)
+@OptionalExpectation
+expect annotation class Config(
+    val sdk: IntArray = [],
+    val minSdk: Int = -1,
+    val maxSdk: Int = -1,
+    val manifest: String = "--default",
+    val application: KClass<out Application> = Application::class,
+    val packageName: String = "",
+    val qualifiers: String = "",
+    val resourceDir: String = "res",
+    val assetDir: String = "assets",
+    val shadows: Array<KClass<*>> = [],
+    val instrumentedPackages: Array<String> = [],
+    val libraries: Array<String> = []
+)
```

Comment:
Robolectric isn't ready for sdk 33 yet, so I added this so we could specify 32 in common tests that use it

We could alternatively just remove our robolectric usage. I think that might be a better practice in general, but I also like having the config in here as a demo for people who want/need it.